index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;600&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <div class="menu-bar">
+      <div class="menu">
+        <div id="bar1" class="bar"></div>
+        <div id="bar2" class="bar"></div>
+        <div id="bar3" class="bar"></div>
+      </div>
+    </div>
+    <!-----------Header section------------->
+    <header>
+      <div class="main-header__brand">
+        <a href="index.html">
+          <img src="images/logo.png" alt="Logo" />
+        </a>
+      </div>
+    </header>
+
+    <section class="main-section">
+      <h1 class="main-section__introduction">I'm Mariana Glad to see you</h1>
+      <h6 class="main-section__text">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </h6>
```

Comment:
- Kindly make sure you use the `<p></p>` tag for this section which is an appropriate tab for the paragraph and it will make your website sematic and make it accessible :grinning:  :grinning:  :grinning:

Reply:
Hello @MaganaAsiati, I do not know what you checked, but I already changed what you pointed out. Please kindly check it again!